cypress/commons/constants/generic/IdConstants.js

```diff
@@ -163,8 +163,12 @@ export const GENERIC_SELECTORS = {
       },
       toolbar: {
         fullscreenButton: '[data-cy=grid-fullscreen-button]',
+        addRowButton: '[data-cy=add-row-button]',
+        deleteRowsButton: '[data-cy=delete-rows-button]',
+        deleteRowsDialogConfirmButton: '[data-cy=delete-row-dialog-confirm-button]',
```

Comment:
Can you change `delete-row` to `delete-rows` to be consistent with the other keys please?

Reply:
DONE